### ISSUE TITLE
Add executor config for spark-submit in doc/spark-on-yarn.md

### ISSUE THREAD

**Issue by shengjh** [zilliztech/arctern#436]
As the doc said, we have a spark cluster with 3 nodes master,slave1,slave2, each equipped with 4 cores and 4 GB memory. Thus, the total memory resources is 12 GB and cores is 12. 
1. We can set --executor-cores=2 to enable each executor have 2 cores, so each node have 2 executors except for the one have AM(because AM will use a executor). 
2. --executor-memory can be derived as 4GB/2=2GB, 2GB-(2GB * 0.1)=1.8G ,we take 1.5G=1536m

In conclusion, we can use 
```shell
spark-submit --num-executors 5 --executors.cores 2 --executor-memory 1536m
```
